project/project.go

```diff
@@ -14,13 +14,12 @@ import (
 )
 
 const (
-	Status200 int = 200
-	Status201 int = 201
-	Status400 int = 400
-	Status401 int = 401
-	Status403 int = 403
-	Status404 int = 404
-	Status500 int = 500
+	EnableContentTrustProjectMetadataKey   MetadataKey = "enable_content_trust"
```

Comment:
Should we prefix the constants with `MetadataKey` instead of using it as a suffix? I think most constants are prefixed with their type name.